cmd/tfplugindocs/testdata/scripts/schema-json/migrate/time_provider_success.txtar

```diff
@@ -0,0 +1,909 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+# Successful run of tfplugindocs -migrate on the Time provider
+[!unix] skip
+
+# Run migrate command
+exec tfplugindocs migrate
+cmpenv stdout expected-output.txt
+
+# Check template files
+cmpenv templates/index.md.tmpl exp-templates/index.md.tmpl
+cmpenv templates/resources/offset.md.tmpl exp-templates/resources/offset.md.tmpl
+cmpenv templates/resources/rotating.md.tmpl exp-templates/resources/rotating.md.tmpl
+cmpenv templates/resources/sleep.md.tmpl exp-templates/resources/sleep.md.tmpl
+cmpenv templates/resources/static.md.tmpl exp-templates/resources/static.md.tmpl
+
+# Check copied non-template files
+cmpenv website/time.erb templates/time.erb
```

Comment:
We should skip migrating these `.erb` files. They are not necessary for provider documentation since it moved from terraform.io to the public registry and it always confuses provider developers. 👍